sunpy/lightcurve/sources/stereo.py

```diff
@@ -0,0 +1,838 @@
+        ax2 = figure.add_subplot(3,1,2)
+        plt.plot_date(dates, data['Bulk Speed [km/s]'].ffill(), '-',label= 'Bulk Speed [km/s]' , color= 'Blue', lw=2)
+        ax3 = figure.add_subplot(3,1,3)
+        plt.plot_date(dates, data['Tkin [deg K]'].ffill(), '-',label= 'Tkin [deg K]' , color='Teal', lw=2)
+        
+        ax1.set_yscale("log",nonposy = "mask")
+        ax1.set_ylabel('[1/cc]')
+        ax1.legend(loc='upper center', bbox_to_anchor=(0.5, 1.2),
+              ncol=1, fancybox=True, shadow=True)
+        ax1.yaxis.grid(True, 'major')
+        ax1.xaxis.grid(False, 'major')
+
+        ax2.set_yscale("log",nonposy = "mask")
+        ax2.set_ylabel('[km/s]')
+        ax2.legend(loc='upper center', bbox_to_anchor=(0.5, 1.2),
+              ncol=1, fancybox=True, shadow=True)
+        ax2.yaxis.grid(True, 'major')
+        ax2.xaxis.grid(False, 'major')
+
+        ax3.set_yscale("log", nonposy = "mask")
+        ax3.set_ylabel('[deg K]')
+        ax3.legend(loc='upper center', bbox_to_anchor=(0.5, 1.2),
+              ncol=1, fancybox=True, shadow=True)
+        ax3.yaxis.grid(True, 'major')
+        ax3.xaxis.grid(False, 'major')
+        ax3.set_xlabel('UTC Time')
+
+        figure.autofmt_xdate()
+        plt.show()
+
+        return figure
+
+
+    @staticmethod
+    def _parse_txt(filepath):
+        """
+        Parses a STEREO PLASTIC file from
+        http://stereo-ssc.nascom.nasa.gov/data/ins_data/plastic/level2/Protons/ASCII/
+
+        """
+        #Reading in Header  
+        with open(filepath) as f:
+            f.readline()
+            header = f.readline().strip('\n').split('\t')
+
+        types = {'sec':'1min', 'date and time': '10min', 'Np [1/cc]': '1hr'}
+
+        #Determining the type of data
+        type_of_data = types.get(header[4])
+        if type_of_data is None:
+            raise ValueError('{var} is not one of the possibilities in the header it \
+                should be one of {poss}'.format(var=header[4], poss= types.keys()))
+
+        
+        #Reading in Data only, using default i.e. 0 value for data_start keyword since all lines before data are commented
+        data = ascii.read(filepath, delimiter = "\s", data_start = 2) 
+       
+        data_modify = []
+        data_modify_other = []
+
+        #Converting separate datetime element into a single datetime.datetime column
+        #Adding the combined datetime column and removing the separate datetime elements columns
+        if type_of_data == '1min':
+            date_and_time_col = data['col7']
+            millisec_col = data['col6']
+            kev_q_time_col = data['col8']
+
+            for i in range(len(data)):
+                data_modify.append((datetime.strptime(str(date_and_time_col[i]) + str(millisec_col[i])[1:], "%Y-%m-%d/%H:%M:%S.%f") ))
+                data_modify_other.append(datetime.strptime(str(kev_q_time_col[i]), "%Y-%m-%d/%H:%M:%S"))
```

Comment:
I don't think you need to convert these to strings... they should be strings already...